FRW/Services/Manager/WalletManager.swift

```diff
@@ -956,6 +956,12 @@ extension WalletManager {
         return accountInfo.storageCapacity - accountInfo.storageUsed < Self.mininumStorageThreshold
     }
 
+    var isBalanceInsufficient: Bool {
+        guard self.isSelectedFlowAccount else { return false }
```

Comment:
It will be an issue, when the flag is checked but the accountInfo is not fetched yet.
So it will always return false

Reply:
That actually happens, but `WalletNewsHandler` is subscribed to the `accountDataDidUpdate` event, and when an update occurs (i.e. when the account has been fetched) the logic is re-evaluated. 
That means that even though the notification doesn't appear immediately, it will after a few seconds.